mobile-sdk/android/customer-secret/java/create-customer-secret.java

```diff
@@ -7,10 +7,9 @@
         return Unit.INSTANCE;
     }));
 
-// On "Pay" tapped
+// On "Pay" clicked
 dojoPaymentFlowHandler.startPaymentFlow(
     new DojoPaymentFlowParams(
         "<paymentIntentId>",
-        "<secret>",// Customer secret key
-        null
+        "<customer secret key>"
```

Comment:
We follow the api response parameters like https://docs.dojo.tech/payments/api#tag/Customers/operation/Customers_CreateCustomerSecret

So <secret> and then a comment is preferred.